articles/getting-started/install-configure-bonsai.md

```diff
@@ -0,0 +1,86 @@
+---
+uid: BonsaiInstallationAndConfiguration
+title: Bonsai Installation and Configuration
+---
+
+# Bonsai Installation and Configuration
+
+## Install Bonsai
+
+To download Bonsai, select between the portable download and the installer download [here](https://bonsai-rx.org/docs/articles/installation.html).
+
+*   The **Portable** download installs a self-contained sandboxed Bonsai. Portable environments enable users to switch between different environments to prevent package conflicts or confusion between similar packages
+
+    *   To install from the **Portable** download, extract the downloaded file. You can start the portable Bonsai by running the `Bonsai.exe` that is inside the extracted folder  
+
+*   The **Installer** download installs Bonsai and all its dependencies globally
+
+    *   To install from the **Installer** download, run the downloaded `Bonsai-X.X.X.exe` file and agree to the involved licenses. You can start the globally installed Bonsai by launching it from the `Bonsai Setup` window after installing or searching for it in your OS's search function, for example
+
+To use Bonsai for interfacing with Open Ephys hardware in a meaningful way, additional packages are required.
+
+<h2 id='open-package-manager'>Open the Bonsai Package Manager</h2>
```

Comment:
This works too, you could have also left it as `## Open the Bonsai Package Manager` and removed the <a> tags.

If you don't have an explicit id, it can then be referenced using `#open-the-bonsai-package-manager`, so taking the entire header, in lowercase, with hyphens instead of spaces